### ISSUE TITLE
Create an .md page for State-level voter registration

### ISSUE THREAD

**Issue by rachelanddata** [Data4Democracy/election-transparency#28]
As part of the voter accessibility project, we would like to collect the voter registration process for each State in an .md file.  For example:

**Colorado**: Online (with drivers license), mail-in, and in-person registration
**Texas**: Register by mail-in application (must have ink signature of applicant) only within county of residence

**_Definition of Done:_** All states (including D.C.) are documented with their voter registration process.

**Comment by epetenko** [comment]
I can take a shot at this for New Jersey, for a start.

**Comment by chrisdick14** [comment]
Awesome! Please edit the [README](https://github.com/Data4Democracy/election-transparency/tree/master/voter-accessibility) and submit a pull request when you add the info.

**Comment by decodyng** [comment]
Question: While we're in the process of gathering this data, might it be better to store in some kind of structured format, instead of just in sentences? Then we could plausibly have a .md file automatically generated off of or alongside that structured data. (Thoughts, @chrisdick14 ?) 

Also, I might take a stab at this, for it has been quiet for 8 days, and I am new to the project and somewhat overwhelmed by the main R thread, but would like to jump on the bus.

**Comment by rachelanddata** [comment]
We can absolutely turn this into a structured format. My thought initially was there would be many different dimensions around voter registration for all states and therefore commonality may be difficult. But, we can always add dimensions to the table as they arise. For now I see a structure like: State and StateAbbr along with indicators: OnlineReg,MailReg,AutoReg,VoterID

**Comment by chrisdick14** [comment]
I agree with @rachelanddata. If you would like to take stab, why don't you start with a structure like she noted, but feel free to add new indicators if it becomes necessary.

**Comment by epetenko** [comment]
I can contribute more states when I have time as well. But I'm wondering if that structured data set doesn't exist online somewhere. I've been hunting.

**Comment by epetenko** [comment]
@I found a list of states that have online voter registration, including links to the registration site: http://www.ncsl.org/research/elections-and-campaigns/electronic-or-online-voter-registration.aspx
I created a quick [Online_registration.xlsx](https://github.com/Data4Democracy/election-transparency/files/781690/Online_registration.xlsx) (sans links) and html table scrape (with links) and will try to get those links into a structured format when I can. 

Edit: Here's the list with links: https://docs.google.com/spreadsheets/d/1TzdHZdkz7z6HEEi3KgPTtllKhAQIsFn47WD0DYtGSkE/edit?usp=sharing

**Comment by markhkim** [comment]
Doesn't this just entail a quick internet search for each state, compiling the results into a csv file, and writing up a script to automatically generate text descriptions? If so, I can get this done in the next few days.

**Comment by chrisdick14** [comment]
Perfect. You can also use the link that epetenko noted above to get started.

**Comment by ghost** [comment]
Hi all, passionate newbie looking to get my feet wet, and was happy to see this data generation issue as it is right up my alley. :-)  I've created a python script that scrapes online and same-day voter registration data from ballotpedia pages, and, it appears that all states have mail in registration as there's a federal mail in voter registration form. 

I'm working on finishing the script up to generate the markup data for each state and output the data to both csv and md for further use, and I hope to be able to submit a pull request within a few days.

I think there's a chance that I got ahead of myself though...was I supposed to get assigned to the issue first?

Sincere apologies for diving in before communicating!

**Comment by ghost** [comment]
Ready for review - https://github.com/Data4Democracy/election-transparency/pull/53
